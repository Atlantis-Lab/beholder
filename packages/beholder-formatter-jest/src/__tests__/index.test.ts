import path from 'path'
import fs from 'fs'
import { format } from '../'

const loadFixtures = name =>
  fs.readFileSync(path.join(__dirname, 'fixtures', `${name}.json`)).toString()

describe('format', () => {
  it('should match base snapshot', async () => {
    const result = await format(loadFixtures('base'))

    expect(result).toMatchSnapshot()
  })
})
