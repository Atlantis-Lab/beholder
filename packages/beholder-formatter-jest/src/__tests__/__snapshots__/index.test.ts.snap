// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`format should match base snapshot 1`] = `
Object {
  "annotations": Array [
    Object {
      "annotation_level": "failure",
      "end_column": 4,
      "end_line": 6,
      "message": "should invalid",
      "path": "/optr/packages/test/src/__tests__/invalid.test.ts",
      "raw_details": "Invalid test run should invalid",
      "start_column": 4,
      "start_line": 6,
      "title": "Invalid test run",
    },
    Object {
      "annotation_level": "failure",
      "end_column": 6,
      "end_line": 11,
      "message": "next should invalid",
      "path": "/optr/packages/test/src/__tests__/invalid.test.ts",
      "raw_details": "Invalid test next next should invalid",
      "start_column": 6,
      "start_line": 11,
      "title": "Invalid test next",
    },
  ],
  "summary": "Found 2 errors",
  "title": "Jest",
}
`;
